test/support/api/dashboards.js

```diff
@@ -1,10 +1,20 @@
 import _ from 'underscore';
-import { getResource } from 'helpers/json-api';
+import { v4 as uuid } from 'uuid';
+
+import { getResource, mergeJsonApi } from 'helpers/json-api';
 
 import fxTestDashboards from 'fixtures/test/dashboards';
 
 const TYPE = 'dashboards';
 
+export function getDashboard(data) {
```

Comment:
We could use `getDashboard()` in the `routeDashboards` command as well, I think? With a sample of `3`?